mappings/net/minecraft/item/FoodComponent.mapping

```diff
@@ -1,37 +1,38 @@
-CLASS axz net/minecraft/item/FoodItemSetting
+CLASS axz net/minecraft/item/FoodComponent
```

Comment:
This suffix conflicts with the recent chat changes. I think this is ok as the chat stuff needs more work.